corehq/apps/api/resources/v0_5.py

```diff
@@ -1170,7 +1168,7 @@ def cursor_query(cls, domain: str, local_timezone: pytz.tzinfo.DstTzInfo, params
 
         with override_settings(USE_TZ=True):
             # TruncDate ignores tzinfo if the queryset is not evaluated within overridden USE_TZ setting
-            return list(queryset[:params['limit']])
+            return list(queryset)
```

Comment:
When is the tastypie limit applied?  Like, is this evaluating the full queryset in memory and then having tastypie return a slice of that list?

Reply:
Oh good point. I'm thinking the queryset is evaluated in memory, and then tastypie returns a slice of that list. That's the only way tastypie would know the count. Going to revert this back to how it was done before

Reply:
[d12d0ed](https://github.com/dimagi/commcare-hq/pull/32922/commits/d12d0edf96eb909c6a385303da112703f90af279)